pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -18,6 +18,9 @@ The initial release of Windows Exporter Metrics contains a single collector avai
 | Key             | Description                                                            | Default   |
 | --------------- | ---------------------------------------------------------------------- | --------- |
 | scrape\_interval | The rate at which metrics are collected from the host operating system | 5 seconds |
+| we.logical\_disk.allow\_disk\_regex | Specify to be scribable regex for logical disk metrics                 | "/.+/"    |
```

Comment:
`scribable` is not really a word, do you mean `allowed`?

Reply:
Yes, I do. The word is what I wanted to say.